language/diem-framework/core/sources/CRSN.move

```diff
@@ -65,7 +65,7 @@ module DiemFramework::CRSN {
 
     const MAX_CRSN_SIZE: u64 = 256;
 
-    public(script) fun allow_crsns(account: &signer) {
+    public fun allow_crsns(account: &signer) {
```

Comment:
I think this probably wants to stay `script` + take an owned `signer` (but @tzakian will know for sure).